.github/workflows/fast.yml

```diff
@@ -148,10 +148,16 @@ jobs:
       uses: ./.github/actions/setup
     - name: Add minimal no_std target
       run: rustup target add thumbv7em-none-eabihf
+    - name: Install ARM bare-metal toolchain (required for blst)
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y gcc-arm-none-eabi
     - name: Build and test codec
       run: cargo build -p commonware-codec --no-default-features --target thumbv7em-none-eabihf --release && du -h target/thumbv7em-none-eabihf/release/libcommonware_codec.rlib && cargo test -p commonware-codec --no-default-features --verbose
     - name: Build utils
       run: cargo build -p commonware-utils --no-default-features --target thumbv7em-none-eabihf --release && du -h target/thumbv7em-none-eabihf/release/libcommonware_utils.rlib
+    - name: Build cryptography
+      run: cargo build -p commonware-cryptography --no-default-features --target thumbv7em-none-eabihf --release && du -h target/thumbv7em-none-eabihf/release/libcommonware_cryptography.rlib
```

Comment:
Can we run the tests with no-default-features?

Reply:
this fails because any library that involves heap allocations seems to require std (because we don't define a custom allocator)

Reply:
We are still running tests with a heap though? Maybe I'm misunderstanding.

Reply:
If you run with no-default-features I guess there's no heap enabled?  maybe we need to add the "alloc" feature. I can try that.

Reply:
```
error: no global memory allocator found but one is required; link to std or add `#[global_allocator]` to a static item that implements the GlobalAlloc trait
```

Reply:
Dug into this, the root issue is that we require the crate-type to be `cdylib`. We need that for WASM compatibility.

There may be a way to work around this but don't think it is worth it atm.